src/documents/tutorial-migratingToInfusion2.0/UIOptionsMigration.md

```diff
@@ -0,0 +1,133 @@
+---
+title: UI Options 2.0 Migration
```

Comment:
This information is general 1.5 to 2.0 changes, not a tutorial specific to UIOptions. It should probably be in a top-level file called "APIChangesFrom1_5To2_0.md"